cloud/aws/metrics/build-scraper

```diff
@@ -2,8 +2,23 @@
 
 # Builds the prometheus metrics scraper Docker image
 
-docker build \
-  -t docker.io/civiform/aws-metrics-scraper:latest \
-  -f metrics_scraper.Dockerfile .
+readonly SHORT_SHA="$(git rev-parse --short HEAD)"
+readonly DATE_IN_UNIX_SECONDS="$(date +%s)"
+readonly SNAPSHOT_TAG="SNAPSHOT-${SHORT_SHA}-${DATE_IN_UNIX_SECONDS}"
+readonly IMAGE="aws-metrics-scraper"
+
+PLATFORM_ARG=()
+if [[ "${PLATFORM}" ]]; then
+  PLATFORM_ARG=(--platform "${PLATFORM}")
+fi
 
-docker push docker.io/civiform/aws-metrics-scraper:latest
+set -e
+set +x
```

Comment:
**suggestion (blocking):** move these to the top of the script, otherwise failures earlier in the script won't stop it executing

Reply:
Updated, thanks!